src/models/SimpleAverage.py

```diff
@@ -1,14 +0,0 @@
-from cleaning_functions import pandas_vector_to_list
```

Comment:
Oh, I really didnt think  of it that way, will revoke it, it's actually a good idea, sorry :+1: